src/qml/nodemodel.cpp

```diff
@@ -37,13 +45,17 @@ void NodeModel::initializeResult([[maybe_unused]] bool success, interfaces::Bloc
 {
     // TODO: Handle the `success` parameter,
     setBlockTipHeight(tip_info.block_height);
+    setVerificationProgress(tip_info.verification_progress);
 }
 
 void NodeModel::ConnectToBlockTipSignal()
 {
     assert(!m_handler_notify_block_tip);
+    assert(!m_verification_progress);
+
```

Comment:
Why this assertion?

Reply:
That should not have been there. I was doing some trying and testing but forgot to remove this statement when I was done.
Thanks for catching this.
Fixed in da06542c7968077d544906ef3e6fbfc56416dcca